[HighlightHalo] add `onHaloAnimationEnd` prop (#1516)
KissKissBankBank/kitten: CHANGELOG.md

## [unreleased]

> - Features: In `HighlightHalo`, new prop `onAnimationEnd` (function to execute when animation is over)

Reviewer: ```suggestion
- Features: Add `onAnimationEnd` to `HighlightHalo` that accepts a function to
  execute when animation is over.
```
